### PR TITLE
Eisenstein series: Small documentation improvement

### PR DESCRIPTION
**Pull request by user202729**
Just some small documentation documentation.

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion. (not aware of one)
- [x] I have created tests covering the changes. (no functional change)
- [x] I have updated the documentation and checked the documentation preview.

--------

Depends on https://github.com/sagemath/sage/pull/38468  to test HTML documentation generation. (But the dependency is not strict, the relevant commits can be cherry-picked out.)

### CODE REVIEW

**Comment by grhkm21** [comment]
What is happening in the `.ci` and `.github` folders?

**Comment by user202729** [comment]
It's some temporary changes (#38468) to make the online HTML documentation render at all. For merging, if #38468 is not merged before this one, it's best to cherry-pick the relevant commits out only.

**Comment by grhkm21** [comment]
Ahh oops I didn't see the commit messages. In that case you should write "Depends on #38468" in your PR description. It is a special keyword in GitHub that will make the dependency show up IIRC

File: src/sage/lfunctions/dokchitser.py [hunk 1]
```diff
@@ -42,6 +42,8 @@
 class Dokchitser(SageObject):
     r"""
     Dokchitser's `L`-functions Calculator.
+    PARI code can be found on
+    `Dokchitser's homepage <https://people.maths.bris.ac.uk/~matyd/computel>`_.
```
Comment (tscrim) [inline src/sage/lfunctions/dokchitser.py]:
```suggestion
    Dokchitser's `L`-functions Calculator.

    PARI code can be found on
    `Dokchitser's homepage <https://people.maths.bris.ac.uk/~matyd/computel>`_.
```

**Review by tscrim — APPROVED** [review]
One very minor thing, but once fixed, positive review.

File: src/sage/lfunctions/dokchitser.py (hunk 1 above)
Comment (user202729) [reply src/sage/lfunctions/dokchitser.py]: Alright, applied. Thanks for the review.

**Comment by tscrim** [comment]
Thank you. Positive review.

**Pull request merged**